examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,72 @@
+"""
+Plotting Datetime Charts
+========================
+
+Plotting datetime charts is handled by :meth:`pygmt.Figure.basemap`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external")`` to display the figure in the default PDF viewer.
+
+    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+    is the desired name and file extension for the saved figure.
```

Comment:
The note can be removed following #1199, since the `Figure.show()` function now works in both python notebooks and interpreters.
```suggestion
```